mkosi/resources/mkosi-initrd/mkosi.conf.d/10-gentoo.conf

```diff
@@ -0,0 +1,74 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+[Match]
+Distribution=gentoo
+
+[Content]
+Packages=
+        sys-apps/systemd
+        sys-apps/kbd
+        app-arch/gzip
+        app-shells/bash
+        
+        sys-fs/bcachefs-tools
+        # libgcc required by bcachefs
+        sys-devel/gcc
+        sys-fs/btrfs-progs
+        sys-fs/e2fsprogs
+        sys-fs/xfsprogs
+        sys-fs/dosfstools
+
+        # Various libraries that are dlopen'ed by systemd
+        dev-libs/libfido2
+        app-crypt/tpm2-tss
+
+        # Useful pkgs
+        sys-apps/less
+        sys-process/procps
+        sys-apps/coreutils
+
+RemovePackages=
+        app-misc/ca-certificates
+        sys-libs/timezone-data
+        sys-kernel/linux-headers
+
+RemoveFiles=
+        # Gentoo Linux doesn't split their gcc-libs package so we manually remove
+        # unneeded stuff here to make sure it doesn't end up in the initrd.
+        /usr/lib/gcc/x86_64-pc-linux-gnu/??/libgfortran.so*
+        /usr/lib/gcc/x86_64-pc-linux-gnu/??/libgo.so*
+        /usr/lib/gcc/x86_64-pc-linux-gnu/??/libgomp.so*
+        /usr/lib/gcc/x86_64-pc-linux-gnu/??/libgphobos.so*
+        /usr/lib/gcc/x86_64-pc-linux-gnu/??/libobjc.so*
+        /usr/lib/gcc/x86_64-pc-linux-gnu/??/libgdruntime.so*
+        /usr/x86_64-pc-linux-gnu/gcc-bin
+        /usr/lib/gcc/x86_64-pc-linux-gnu/??/include
+        /usr/share/gcc-data
+        /usr/share/gdb
+        /usr/lib/gcc/x86_64-pc-linux-gnu/??/install-tools
+        /usr/lib/gcc/x86_64-pc-linux-gnu/??/plugin
+        /usr/x86_64-pc-linux-gnu/binutils-bin
+        /usr/libexec/gcc/x86_64-pc-linux-gnu
+        /usr/lib/gcc/x86_64-pc-linux-gnu/??/*.a
+
+        # Remove all files that are only required for development.
+        /usr/lib/*.a
+        /usr/lib64/*.a
+        /usr/include/*
+        /usr/lib64/pkgconfig
+        /usr/lib/debug
+     
+        # Portage
```

Comment:
initrd should idealy be `usr-only`.
this needs a better fix in `emerge.py`... we keep `/etc/portage` in final image as a way of documenting what was used with `mkosi`; i don't have a good solution for this yet.